sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/MetricsAdvisorClient.cs

```diff
@@ -66,644 +56,316 @@ public MetricsAdvisorClient(Uri endpoint, TokenCredential credential)
         {
         }
 
-        /// <summary>
-        /// Initializes a new instance of the <see cref="MetricsAdvisorClient"/> class.
-        /// </summary>
-        /// <param name="endpoint">The endpoint to use for connecting to the Metrics Advisor Cognitive Service.</param>
-        /// <param name="credential">A credential used to authenticate to the service.</param>
-        /// <param name="options">A set of options to apply when configuring the client.</param>
-        /// <exception cref="ArgumentNullException"><paramref name="endpoint"/> or <paramref name="credential"/> is null.</exception>
-        public MetricsAdvisorClient(Uri endpoint, TokenCredential credential, MetricsAdvisorClientsOptions options)
-        {
-            Argument.AssertNotNull(endpoint, nameof(endpoint));
-            Argument.AssertNotNull(credential, nameof(credential));
-
-            options ??= new MetricsAdvisorClientsOptions();
-
-            _clientDiagnostics = new ClientDiagnostics(options);
-            HttpPipeline pipeline = HttpPipelineBuilder.Build(options, new BearerTokenAuthenticationPolicy(credential, Constants.DefaultCognitiveScope));
-
-            _serviceRestClient = new MicrosoftAzureMetricsAdvisorRestAPIOpenAPIV2RestClient(_clientDiagnostics, pipeline, endpoint.AbsoluteUri);
-        }
-
-        /// <summary>
-        /// Initializes a new instance of the <see cref="MetricsAdvisorClient"/> class. This constructor
-        /// is intended to be used for mocking only.
-        /// </summary>
-        protected MetricsAdvisorClient()
-        {
-        }
-
-        #region TimeSeries
```

Comment:
It looks like the order of TimeSeries and MetricFeedback region has changed in the new version - would it be possible to keep those in the same order as the original file so it is easier to read the diff?